### PR TITLE
Add link to request for library update and report outdated library

### PR DESCRIPTION
**Pull request by sufuf3**
cc #145
Add a request update link on library page

image

click link and redirect to GitHub page

image

### CODE REVIEW

File: templates/library.html [hunk 1]
```diff
@@ -38,7 +38,7 @@ <h1 style="margin-top: 00px;" id="libraryName">{{library.name}}</h1>
           <script type="text/javascript" src="https://cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=cdnjscom" id="_carbonads_js" defer></script>
         </div>
       </div>
-       <div class="input-group version-dropdown">
+        <div class="input-group version-dropdown">
```
Comment (PeterDaveHello) [inline templates/library.html]: Unrelated diff ...

**Comment by PeterDaveHello** [comment]
Should give a screenshot and example link.

**Comment by PeterDaveHello** [comment]
Wrong syntax in commit message.

**Comment by sufuf3** [comment]
example link: /libraries/jquery

**Comment by PeterDaveHello** [comment]
1) You didn't provide the link
2) You should provide it in the first comment
3) The design of the place and style of the link ... makes itself looks like a broken, I don't think we should make it appear in that place and in that way.

**Comment by sufuf3** [comment]
@cdnjs/intern3 Please help me review this PR, thanks.

I change this request update link place on library page
image


And redirect to GitHub page as following:
image

If there is any suggest, please let me know.

**Comment by PeterDaveHello** [comment]
It's a bad idea to simply clone the template from new library request, totally different purpose.

**Comment by sufuf3** [comment]
@PeterDaveHello I pulled from this repo. , please help me review this PR.
And if there is any suggest, please let me know.
Thank you.

**Comment by PeterDaveHello** [comment]
I mean the issue template, it's originally for new library request, you should not just copy and paste as the purpose is different.

**Comment by sufuf3** [comment]
What items should issue template take is better?
How about as following? (`XXXXX` will auto add from library API.)

**Library name:** XXXXX
**CDNJS version:** XXXXX
**CDNJS URL:** XXXXX
**Git repository url:** XXXXX
**npm package name or url (if there is one):** XXXXX
**Need update items and description:**
 * [ ] **description:** XXXXX -> _____________________
 * [ ] **version:**   XXXXX -> _____________________
 * [ ] **author:**    XXXXX -> _____________________
 * [ ] **filename:**     XXXXX -> _____________________
 * [ ] **homepage:**     XXXXX -> _____________________
 * [ ] **repository:**     XXXXX -> _____________________
 * [ ] **keywords:**     XXXXX -> _____________________
 * [ ] **license:**     XXXXX -> _____________________

**Wanna say something? Leave message here:** 

If there is any suggest, please let me know.

**Comment by PeterDaveHello** [comment]
@sufuf3 It's much more better that you started to think about its content now, and you should also explain the reason why or why not the items would be or would not be listed here.

**Comment by sufuf3** [comment]
I refer to `package.json`.
In **Need update items and description:**, I include all of the items, exclude as following:
1. `repository`, because it show in **Git repository URL:** 
2. `name`, it show in **Library name:**
3. `auto-update`, this item is hard to detect, cause it may not exist, from npm to git repo., or from git repo, to npm. I think it can let the user leave a message.

If there is any suggest, please let me know.
Thank you.

**Comment by sufuf3** [comment]
Or how about as following? (`XXXXX` will auto add from library API.)

**Library name:** XXXXX
**CDNJS version:** XXXXX
**CDNJS URL:** XXXXX
**Git repository url:** XXXXX
**npm package name or url (if there is one):** XXXXX
**Need update items and description:**
 * [ ] **description:** XXXXX -> _____________________
 * [ ] **version:**   XXXXX -> _____________________
 * [ ] **author:**    XXXXX -> _____________________
 * [ ] **filename:**     XXXXX -> _____________________
 * [ ] **homepage:**     XXXXX -> _____________________
 * [ ] **repository:**     XXXXX -> _____________________
 * [ ] **keywords:**     XXXXX -> _____________________
 * [ ] **license:**     XXXXX -> _____________________
 * [ ] **auto-update:** 
   * [ ] None to npm auto-update
   * [ ] None to git auto-update
   * [ ] git auto-update to npm auto-update
   * [ ] npm auto-update to git auto-update

**Wanna say something? Leave message here:**

**Comment by sufuf3** [comment]
@hare1039 Could you give me some suggestion and make this PR to be better?

**Comment by clairetsai818** [comment]
**- [ ] This library is not updated to the latest version

- [ ] The GitHub repo has changed

- [ ] Some info of this lib needs to be updated
       Info needed to be updated:**

I think that we can list out the reasons that happens often, so that the issuer only needs to check that box.

**Comment by sufuf3** [comment]
@clairetsai818 Thank you for your suggestion. I modified as the following, hope it is okay.

**Library name:** XXXXX
**CDNJS version:** XXXXX
**CDNJS URL:** XXXXX
**Git repository url:** XXXXX
**npm package name or url (if there is one):** XXXXX

**Update Reason:**
 * [ ] **This library is not updated to the latest version.**
 * [ ] **The GitHub repo has changed.**
 * [ ] **Some info of this lib needs to be updated:**

   * [ ] **description:** XXXXX -> _____________________
   * [ ] **version:**   XXXXX -> _____________________
   * [ ] **author:**    XXXXX -> _____________________
   * [ ] **filename:**     XXXXX -> _____________________
   * [ ] **homepage:**     XXXXX -> _____________________
   * [ ] **repository:**     XXXXX -> _____________________
   * [ ] **keywords:**     XXXXX -> _____________________
   * [ ] **license:**     XXXXX -> _____________________

**Wanna say something? Leave message here:**

**Comment by hare1039** [comment]
> This library is not updated to the latest version.

A little bit too long. How about `This library needs update to latest version`
> The GitHub repo has changed.

Not necessary GitHub I think.

Will those `XXXXX` auto filled?

**Comment by sufuf3** [comment]
I modified it.
And I think the redirect url to the GitHub page template maybe can using as the following: 

**Library name:** XXXXX
**CDNJS version:** XXXXX
**CDNJS URL:** XXXXX
**Git repository url:** XXXXX
**npm package name or url (if there is one):** XXXXX

**Update Reason:**
   * [ ] **description:** XXXXX -> _____________________
   * [ ] **version:**   XXXXX -> _____________________
   * [ ] **author:**    XXXXX -> _____________________
   * [ ] **filename:**     XXXXX -> _____________________
   * [ ] **homepage:**     XXXXX -> _____________________
   * [ ] **repository:**     XXXXX -> _____________________
   * [ ] **keywords:**     XXXXX -> _____________________
   * [ ] **license:**     XXXXX -> _____________________

**Wanna say something? Leave message here:** 

----

I refer to `package.json`.
In **Need update items and description:**, I include all of the items, exclude as following:
1. `repository`, because it show in **Git repository URL:** 
2. `name`, it show in **Library name:**
3. `auto-update`, this item is hard to detect, cause it may not exist, from npm to git repo., from git repo. to npm or file structure problem. I think it can let the user leave a message. And we get **Git repository url** and **npm package name or url** info, it let us know faster.
3.  XXXXX will auto add from library API.

If there is any suggest, please let me know.
Thank you.

**Comment by gocreating** [comment]
It will be better readability with indention

**Comment by sufuf3** [comment]
@gocreating thanks, I'll try to modify this.
sorry for late response.

**Comment by qas612820704** [comment]
Hi @sufuf3, it seems like #71 can be done by this PR at the same time.


By modifying to @gocreating 's code like
```js
{{
  let getURL = (type, ...args) => {
    if (type == 'update')
      return `https://github.com/cdnjs/cdnjs/issues/new?title=${...}&body=${...}&...`;
    else if (type == 'report')
      return ...;
   };
}}
<a href="{{ getURL('update', ...) }}">
  request update
</a>
<a href="{{ getURL('report', ...) }}">
  report lib
</a>
```

How about this? :)

**Comment by sufuf3** [comment]
@qas612820704 Thank you very much.
I'll try to use it.

**Comment by sufuf3** [comment]
@PeterDaveHello we added two links as the followings:
1. request for library update #145 
2. report outdated library #71 

- This is the page show the links:
image
- This is the link redirect to GitHub to request for library update:
image
- This is the link redirect to GitHub to report outdated library:
image


If you have any suggestion, please let me know.
Thank you.

**Comment by PeterDaveHello** [comment]
We don't need so many items, just its version.
Also I don't think we need user to fill repo or hosted version info.

**Comment by sufuf3** [comment]
I modified as the followings.
If you have any suggestion, please let me know.
Thanks.

* This is the page show the links:
image
* This is the link redirect to GitHub to request for library update:
image
* This is the link redirect to GitHub to report outdated library:
image

**Comment by PeterDaveHello** [comment]
It's confused about what's the difference from the name of the title/button, also, I don't like the place where they are placed, the button(s) should be individual, not mixed with the dropdown list.

**Comment by PeterDaveHello** [comment]
This situation is not good to use `wanna say something`, it's about the description of what the request is.

**Comment by sufuf3** [comment]
I modified as the followings.
If you have any suggestion, please let me know.
Thanks.

- modified the button part
image

-  remove `wanna say something` part of the description.
image
